Source/WebCore/editing/TextIterator.h

```diff
@@ -274,6 +274,7 @@ class BackwardsCharacterIterator {
     bool atEnd() const { return m_underlyingIterator.atEnd(); }
     void advance(int numCharacters);
 
+    StringView text() const { return m_underlyingIterator.text().substring(0, m_underlyingIterator.text().length() - m_runOffset); }
```

Comment:
Optional: Since your start index is always `0`, you could also use `StringView::left()`

```suggestion
    StringView text() const { return m_underlyingIterator.text().left(m_underlyingIterator.text().length() - m_runOffset); }
```